doc/installationguide.md

```diff
@@ -156,12 +156,14 @@ have implications in the use of attributes with Context Providers, so this flag
 being collected by the device, the expiration daemon will reclaim it. This attribute is optional (if it doesn't exist, commands won't expire).
 * **pollingDaemonFrequency**: time between collection of expired commands in milliseconds. This attribute is optional
 (if this parameter doesn't exist the polling daemon won't be started).
- * **autocast**: When enabled, the IoT Agents will try to cast attribute's values considering the JSON native type (only for NGSIv2).
+* **autocast**: When enabled, the IoT Agents will try to cast attribute's values considering the JSON native type (only for NGSIv2).
 * **multiCore**: When enabled, the IoT Agents runs in multi-thread environment to 
 take advantage of multi-core systems. It allows two values `true` or `false`. This attribute 
 is optional with default to false, which means that the IoTAgent runs in a single thread.
 For more details about multi-core functionality, please refer to the 
 [Cluster](https://nodejs.org/api/cluster.html) module in Node.js and [this section](howto.md#iot-agent-in-multi-thread-mode) of the library documentation.
+* **defaultExpressionLanguage**: the default expression language used to compute expressions, possible values are: `legacy|jexl`. When not set or wrongly set,
```

Comment:
```suggestion
* **defaultExpressionLanguage**: the default expression language used to compute expressions, possible values are: `legacy`or `jexl`. When not set or wrongly set,
```

(This is the usual style in this piece of documentation to mean optionalitty in a configuration parameter values)

Reply:
will fix

Reply:
fix by 8eb5505